### ISSUE TITLE
How to reverse port forwarding

### ISSUE THREAD

**Issue by chaudhryfaisal** [myzhang1029/penguin-rs#24]
Hi there, you mentioned there is no reverse port forwarding because it is equivalent to spawning another server on the client side

I have a client behind the router that connects to the remote server, how can I expose a local SSH port on a remote server?
server does not have a way to connect to the client so how would launching another server on the client side would help this?

Thanks

**Comment by myzhang1029** [comment]
You are absolutely right. Now that you think about it, the "equivalence" part isn't true and reverse port forwarding is really required to expose a local port behind a NAT.

Honestly, I was just lazy to think of a way to fit this feature in the current design. I'll keep that in mind and maybe implement it one day. In the meantime I'll update `README` and if you or anyone else have good solutions, feel free to put it here.

**Comment by myzhang1029** [comment]
- [x] Mux protocol support (#51)
- [ ] Server support
- [ ] Client support
